openc3/lib/openc3/system/system.rb

```diff
@@ -92,6 +92,18 @@ def self.setup_targets(target_names, base_dir, scope:)
               zip_file.extract(entry, path) unless File.exist?(path)
             end
           end
+
+          # Now add any modifications in targets_modified/TARGET/cmd_tlm
+          # This adds support for remembering dynamically created packets
+          # target.txt must be configured to either use all files in cmd_tlm folder (default)
+          # or have a predetermined empty file like dynamic.txt
+          bucket_path = "#{scope}/targets_modified/#{target_name}/cmd_tlm"
+          dirs, files = bucket.list_files(bucket: ENV['OPENC3_CONFIG_BUCKET'], path: bucket_path)
+          files.each do |file|
+            bucket_key = File.join(bucket_path, file['name'])
+            local_path = "#{base_dir}/targets/#{target_name}/cmd_tlm/#{file['name']}"
+            bucket.get_object(bucket: ENV['OPENC3_CONFIG_BUCKET'], key: bucket_key, path: local_path)
+          end
```

Comment:
This block is pulling down the dynamically created packets back into the local target_modified directory?

Reply:
It is pulling them down into the files used to create the System object yes.